course04/homework00/serhan/sarray/sarray.cc

```diff
@@ -0,0 +1,42 @@
+#include "sarray.h"
+
+sarray::sarray(size_t maxSize) :
+        mCapacity{maxSize}
+{
+}
+
+sarray::sarray()
+{
+    mCapacity = 32;
+}
+
+void sarray::push_back(int element)
+{
+    if (size() + 1 > capacity())
```

Comment:
Its correct but I would have compared to max_size. capacity here is equal to max_size but thats only because this sarray is fixed size